chrome/skin/default/zotero/itemPane.css

```diff
@@ -67,3 +67,29 @@
 {
 	min-height: 20px;
 }
+
+#zotero-feed-item-toggleRead-button {
+	overflow: hidden;
+	text-overflow: ellipsis;
+	max-width: 150px;
+}
+
+#zotero-feed-item-addTo-button {
+	max-width: 250px;
+}
+
+#zotero-feed-item-addTo-button button {
+	overflow: hidden;
+	text-overflow: ellipsis;
+}
+
+/* Set to hidden in user-agent css for some reason. */
+#zotero-feed-item-addTo-button .button-icon {
+	display: block;
+	margin-right: 5px
+}
+
+/* Set to hidden in user-agent css for some reason. */
+#zotero-feed-item-addTo-button .menu-iconic-left {
+	visibility: visible;
+}
```

Comment:
Remove these — set class `menuitem-iconic`/`menu-iconic` on the menuitem/menu instead.

https://developer.mozilla.org/en-US/docs/Mozilla/Tech/XUL/menuitem#Style_classes
https://developer.mozilla.org/en-US/docs/Mozilla/Tech/XUL/menu#Style_classes